pkg/install/install.go

```diff
@@ -156,6 +157,19 @@ func Install(ctx context.Context, image string, opts *Options) error {
 		return err
 	}
 
+	// Validate the pod CIDRsq
+	var errs []error
+	for _, cidr := range finalConfForValidation.PodCIDRs {
+		if cidr != "" {
```

Comment:
If a `cidr` is given as `""`, do we want to produce an error?

Reply:
No, in case the user is overwriting a previously provided CIDR to make it empty again.

Reply:
Is that possible? These cidrs are provided as a slice, so a user couldn't just delete one. They would have to provide them all again, minus the one they are trying to remove.

Reply:
Yeah you're right. But if they want to get rid of all of the CIDRs that have been provided already I do think they have to pass the empty string.